crates/autopilot/src/solvable_orders.rs

```diff
@@ -304,6 +308,9 @@ impl SolvableOrdersCache {
         };
 
         tracing::debug!(%block, "updated current auction cache");
+        self.metrics
```

Comment:
GIven that we know that this takes a significant amount of time we could already add metrics for the individual stages of the auction building.
I assume most of the time will likely be spent on the DB query but there will probably also be outliers in the individual steps that need to be ironed out.

Reply:
Added a HistogramVec for individual update stages.